mkdocs.yml

```diff
@@ -53,7 +54,7 @@ plugins:
         account_management/xias_guest.md: account_management/xias/guest_instructions.md
         account_management/xias_sites.md: account_management/xias/pi_site_management.md
         account_management/xias_users.md: account_management/xias/pi_guest_management.md
-        cheaha/conda.md: cheaha/software/software.md#anaconda-on-cheaha
+        cheaha/conda.md: cheaha/software/software.md/#anaconda-on-cheaha
```

Comment:
Please revert this change. If I suggested it that was my mistake. It works the way it was before, we were entering the url incorrectly when we tested.

Reply:
Fixed !